org.rcpml.core/src/org/rcpml/core/RCPML.java

```diff
@@ -33,4 +36,27 @@ public static IController create(Document document) {
 	public static IController createWithConstructor(Document document) {
 		return new Controller(document, true);
 	}
+	
+	public static Composite createComposite(Document doc, Composite parent) {
+		Object presentation = RCPML.renderDocument(doc);
```

Comment:
[minor] Get rid of Object usage (use common interfaces)

Reply:
The only one common class for presentation is Object. In some cases getPresentation() method returns Comrosite objects, in other implementors of ICompositeHolder, but also it can return IDataSource, Application, just null, IPerspectiveFactory, etc.